WalletWasabi.Documentation/ClientRelease/ReleaseNotesTemplate.md

```diff
@@ -0,0 +1,42 @@
+### _[Wasabi Wallet](https://WasabiWallet.io) is an easy to use, privacy-focused, open-source, non-custodial, Bitcoin wallet_
+
+# Download
+:window: [Windows](https://github.com/zkSNACKs/WalletWasabi/releases/download/v2.0.x/Wasabi-2.0.x.msi)
```

Comment:
I would keep these link empty.
If the release is silent the format would be `v2.0.x.x` and not `v2.0.x`, and if there is a hardfork it would be `v2.x.x`.

Reply:
It's easier to change a letter than to search for the link elsewhere.

If we do a hardfork we can update this template to 2.1.x